galasa-ui/src/app/actions/getUserAccessTokens.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import { AuthenticationAPIApi, AuthTokens } from "@/generated/galasaapi";
+import { createAuthenticatedApiConfiguration } from "@/utils/api";
+import * as Constants from "@/utils/constants";
+
+const apiConfig = createAuthenticatedApiConfiguration();
```

Comment:
If the user's JWT expires, will a new API client be created or will this still use the old JWT?